examples/QuickJournal/QuickJournal/Services/NavigationService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Threading.Tasks;
+using QuickJournal.ViewModels;
+using QuickJournal.Views;
+using Xamarin.Forms;
+
+namespace QuickJournal.Services
+{
+    public static class NavigationService
+    {
+        static INavigation Navigation => Application.Current.MainPage?.Navigation;
+
+        public static async Task NavigateTo(BaseViewModel viewModel)
+        {
+            if (Navigation == null)
+                throw new InvalidOperationException("Main page does not support navigation");
```

Comment:
I remember seeing a comment in one of my PRs about brackets. Do we have a consistent opinion about that?

Reply:
So.... My style was to avoid brackets for one liners, and so I went with my style because I took inspiration from another project I wrote ;)
But you're right, that in Realm we generally put brackets everywhere

Reply:
Not trying to bring in too much of my opinion, since consistency is the most important part for me here but I'm exactly the other way round because brackets make sure that when you add another statement and forget to add the brackets you don't alter the whole thing.

But as I said, my recommendation is to go with whatever our usual, recommended, consistent code style would be. :)

Reply:
That is also Nikola's opinion 😁
Anyways to be consistent I also put brackets everywhere, so we follow the style of the main project